articles/specs/launch.0.1.1.xsd

```diff
@@ -524,8 +526,7 @@
         <xs:element ref="param"/>
         <xs:element ref="remap"/>
       </xs:choice>
-    </xs:complexType>
-    <xs:attribute name="package" type="xs:string" use="required">
+          <xs:attribute name="package" type="xs:string" use="required">
```

Comment:
The indentation seems odd here.

Reply:
It is. Fixed in 25db0215cb72dc84d9828dd25895593512f57582.